iOS/FlowWalletKit/Sources/Keys/PrivateKey.swift

```diff
@@ -125,6 +125,11 @@ public class PrivateKey: KeyProtocol {
         return pubK.uncompressed.data.dropFirst()
     }
 
+    public func privateKey(signAlgo: Flow.SignatureAlgorithm = .ECDSA_P256) throws -> String? {
+        let privateKey = pk.data.hexValue
```

Comment:
Private key can generate different key with different sign curve. Refer to seed phrase, use switch case instead return raw key